cluster-autoscaler/cloudprovider/azure/azure_util.go

```diff
@@ -75,6 +75,9 @@ const (
 	k8sWindowsVMAgentPoolPrefixIndex       = 1
 	k8sWindowsVMAgentOrchestratorNameIndex = 2
 	k8sWindowsVMAgentPoolInfoIndex         = 3
+
+	nodeLabelTagName = "k8s.io|cluster-autoscaler|node-template|label|"
```

Comment:
Due to the restrictions on using slashes in the tag names, I decided to go with pipes for the separator here

Reply:
I believe using underscores would make more sense.

Reply:
I agree and will be more of a common convention, I'll update these